.github/workflows/ci-build.yml

```diff
@@ -39,18 +39,17 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - name: Install Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
       with:
-        go-version: 1.17.x
+        go-version: 1.18.x
         
     - name: Checkout code
       uses: actions/checkout@v2
     
     - name: Lint
-      uses: golangci/golangci-lint-action@v2
+      uses: golangci/golangci-lint-action@v3
       with:
-        version: v1.43.0
-        skip-go-installation: true # we already installed Go with `actions/setup-go@v2` above
```

Comment:
was removing this `skip-go-installation` intentional?

Reply:
yes, it was not needed with  golangci/golangci-lint-action@v3